napari/_tests/test_examples.py

```diff
@@ -44,6 +44,8 @@ def qapp():
 @pytest.mark.parametrize("fname", examples)
 def test_examples(qapp, fname, monkeypatch, capsys):
     """Test that all of our examples are still working without warnings."""
+    if fname == "embed_ipython.py":
```

Comment:
Do we need this? Can it come in it's own PR - it seem unrelated

Reply:
This is to prevent test fail when locally run of pytest

Reply:
This should have been skipped already by line 16 above... is it not skipping for you?  have you merged master?

Reply:
It looks like it was solved after I start this PR. But now my change is obsolete.